android/java/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java

```diff
@@ -311,28 +316,23 @@ public void updateBookmarkButton(boolean isBookmarked, boolean editingAllowed) {
 
     @Override
     public boolean onLongClick(View v) {
-        String description = "";
-        Resources resources = mContext.getResources();
-
         if (v == mHomeButton) {
             // It is currently a new tab button when homepage is disabled.
             if (!HomepageManager.isHomepageEnabled()) {
                 TabUtils.showTabPopupMenu(mContext, v);
                 return true;
             }
 
-            description = resources.getString(R.string.accessibility_toolbar_btn_home);
```

Comment:
i think we should keep the toast for other options like home.

Reply:
It is mentioned in the issue ticket that we should remove the tooltips for the other toolbar icons that don't have functionality. so should we keep the toast?